python.d/elasticsearch.chart.py

```diff
@@ -166,12 +190,74 @@
             ['indexing_latency', 'indexing', 'absolute', 1, 1000],
             ['flushing_latency', 'flushing', 'absolute', 1, 1000]
         ]},
+    'index_translog_operations': {
+        'options': [None, 'Translog Operations', 'count', 'translog',
+                    'elastic.index_translog_operations', 'area'],
+        'lines': [
+            ['indices_translog_operations', 'total', 'absolute'],
+            ['indices_translog_uncommitted_operations', 'uncommited', 'absolute']
+        ]},
+    'index_translog_size': {
+        'options': [None, 'Translog Size', 'MB', 'translog',
+                    'elastic.index_translog_size', 'area'],
+        'lines': [
+            ['indices_translog_size_in_bytes', 'total', 'absolute', 1, 1048567],
+            ['indices_translog_uncommitted_size_in_bytes', 'uncommited', 'absolute', 1, 1048567]
+        ]},
+    'index_segments_count': {
+        'options': [None, 'Total Number Of Indices Segments', 'count', 'indices segments',
+                    'elastic.index_segments_count', 'line'],
+        'lines': [
+            ['indices_segments_count', 'segments', 'absolute']
+        ]},
+    'index_segments_memory': {
+        'options': [None, 'Indices Segments Memory Usage', 'MB', 'indices segments',
+                    'elastic.index_segments_memory', 'area'],
+        'lines': [
+            ['indices_segments_memory_in_bytes', 'total', 'absolute', 1, 1048567],
```

Comment:
about this
i suggest
1. remove `total` from `index_segments_memory` chart
2. if total is needed as chart please add another one

about `index_segments_memory` are you sure about `area`? maybe `stacked`?

Reply:
Yes, this part definitely needed some adjustments.
It appears that the most logical way to do this would be: 
1. Create an `index_segments_memory` stacked chart that has every metric from `indices.segments.` except for `indices.segments.index_writer_memory_in_bytes`. This is the total amount of memory needed to support all existing segments.
2. Create a separate chart with just `indices.segments.index_writer_memory_in_bytes`: this shows the amount of data held in Lucene's `IndexWriter` and not yet written to new on-disk segments.

This way the `total` metric is unnecessary.
I've pushed the necessary changes.